[package] openssl/1.1.1h: can't build on windows

<!-- 
  Please don't forget to update the issue title.
  Include all applicable information to help us reproduce your problem.
-->

### Package and Environment Details (include every applicable attribute)
  * Package Name/Version: **openssl/1.1.1h:**
  * Operating System+version: **Windows 10**
  * Compiler+version: **MSVC**
  * Conan version: **conan 1.32.1**
  * Python version: **Python 3.8.5**


### Conan profile (output of `conan profile show default` or `conan profile show <profile>` if custom profile is in use)
```
[settings]
arch=x86_64
arch_build=x86_64
build_type=Debug
compiler=Visual Studio
compiler.cppstd=14
compiler.runtime=MD
compiler.version=16
os=Windows
os_build=Windows
[options]
*:fPIC=True
*:with_vulkan=True
[build_requires]
[env]
```


### Steps to reproduce (Include if Applicable)
conan install openssl/1.1.1h@ --build outdated


### Logs (Include/Attach if Applicable)
<details><summary>Click to expand log</summary>

```
openssl/1.1.1h: WARN: Package binary is corrupted, removing: 402cf37ea699030d7c8abcaa214949592ce22e9b
nasm/2.14: Package is up to date
strawberryperl/5.30.0.1: Package is up to date
Installing package: openssl/1.1.1h
Requirements
    openssl/1.1.1h from 'conan-center' - Cache
Packages
    openssl/1.1.1h:402cf37ea699030d7c8abcaa214949592ce22e9b - Build
Build requirements
    nasm/2.14 from 'conan-center' - Cache
    strawberryperl/5.30.0.1 from 'conan-center' - Cache
Build requirements packages
    nasm/2.14:01edd76db8e16db9b38c3cca44ec466a9444c388 - Cache
    strawberryperl/5.30.0.1:456f15897172eef340fcbac8a70811f2beb26a93 - Cache

Installing (downloading, building) binaries...
nasm/2.14: Already installed!
nasm/2.14: Appending PATH environment variable: C:\Users\malas\.conan\data\nasm\2.14\_\_\package\01edd76db8e16db9b38c3cca44ec466a9444c388\bin
strawberryperl/5.30.0.1: Already installed!
strawberryperl/5.30.0.1: Appending PATH environment variable: C:\.conan\1be2a02\1\bin
openssl/1.1.1h: Applying build-requirement: strawberryperl/5.30.0.1
openssl/1.1.1h: Applying build-requirement: nasm/2.14
openssl/1.1.1h: WARN: Build folder is dirty, removing it: C:\Users\malas\.conan\data\openssl\1.1.1h\_\_\build\402cf37ea699030d7c8abcaa214949592ce22e9b
openssl/1.1.1h: Copying sources to build folder
openssl/1.1.1h: Building your package in C:\Users\malas\.conan\data\openssl\1.1.1h\_\_\build\402cf37ea699030d7c8abcaa214949592ce22e9b
openssl/1.1.1h: Generator txt created conanbuildinfo.txt
openssl/1.1.1h: Calling build()
**********************************************************************
** Visual Studio 2019 Developer Command Prompt v16.6.2
** Copyright (c) 2020 Microsoft Corporation
**********************************************************************
[vcvarsall.bat] Environment initialized for: 'x64'
openssl/1.1.1h: using target: VC-conan-Debug-Windows-x86_64-Visual Studio-16 -> VC-WIN64A
openssl/1.1.1h: my %targets = (
    "VC-conan-Debug-Windows-x86_64-Visual Studio-16" => {
        inherit_from => [ "VC-WIN64A" ],
        cflags => add("-Zi -Ob0 -Od -MD"),
        cxxflags => add("-Zi -Ob0 -Od -MD"),

        includes => add("C:/.conan/1be2a02/1/include"),
        lflags => add(""),








    },
);

openssl/1.1.1h: ['"VC-conan-Debug-Windows-x86_64-Visual Studio-16"', 'no-shared', '--prefix="C:\\Users\\malas\\.conan\\data\\openssl\\1.1.1h\\_\\_\\package\\402cf37ea699030d7c8abcaa214949592ce22e9b"', '--openssldir="C:\\Users\\malas\\.conan\\data\\openssl\\1.1.1h\\_\\_\\package\\402cf37ea699030d7c8abcaa214949592ce22e9b\\res"', 'no-unit-test', 'threads', 'PERL=C:\\.conan\\1be2a02\\1\\bin\\perl.exe', 'no-tests', '--debug']
WARN: replace_in_file didn't find pattern '/MTd ' in 'Configurations\10-main.conf' file.
WARN: replace_in_file didn't find pattern '/MTd"' in 'Configurations\10-main.conf' file.
Configuring OpenSSL version 1.1.1h (0x1010108fL) for VC-conan-Debug-Windows-x86_64-Visual Studio-16
Using os-specific seed configuration
Creating configdata.pm
Creating makefile

**********************************************************************
***                                                                ***
***   OpenSSL has been successfully configured                     ***
***                                                                ***
***   If you encounter a problem while building, please open an    ***
***   issue on GitHub <https://github.com/openssl/openssl/issues>  ***
***   and include the output from the following command:           ***
***                                                                ***
***       perl configdata.pm --dump                                ***
***                                                                ***
***   (If you are new to OpenSSL, you might want to consult the    ***
***   'Troubleshooting' section in the INSTALL file first)         ***
***                                                                ***
**********************************************************************

Microsoft (R) Program Maintenance Utility Version 14.26.28806.0
Copyright (C) Microsoft Corporation.  All rights reserved.

        "C:\.conan\1be2a02\1\bin\perl.exe" "-I." -Mconfigdata "util\dofile.pl"  "-omakefile" "include\crypto\bn_conf.h.in" > include\crypto\bn_conf.h
        "C:\.conan\1be2a02\1\bin\perl.exe" "-I." -Mconfigdata "util\dofile.pl"  "-omakefile" "include\crypto\dso_conf.h.in" > include\crypto\dso_conf.h
        "C:\.conan\1be2a02\1\bin\perl.exe" "-I." -Mconfigdata "util\dofile.pl"  "-omakefile" "include\openssl\opensslconf.h.in" > include\openssl\opensslconf.h
        "C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.26.28801\bin\HostX64\x64\nmake.exe" /                   depend && "C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.26.28801\bin\HostX64\x64\nmake.exe" /                   _all

Microsoft (R) Program Maintenance Utility Version 14.26.28806.0
Copyright (C) Microsoft Corporation.  All rights reserved.


Microsoft (R) Program Maintenance Utility Version 14.26.28806.0
Copyright (C) Microsoft Corporation.  All rights reserved.

        cl  /Zi /Fdossl_static.pdb /MD /Zl /Gs0 /GF /Gy -Zi -Ob0 -Od -MD /W3 /wd4090 /nologo /Od /I "." /I "include" -D"L_ENDIAN" -D"OPENSSL_PIC" -D"OPENSSL_CPUID_OBJ" -D"OPENSSL_IA32_SSE2" -D"OPENSSL_BN_ASM_MONT" -D"OPENSSL_BN_ASM_MONT5" -D"OPENSSL_BN_ASM_GF2m" -D"SHA1_ASM" -D"SHA256_ASM" -D"SHA512_ASM" -D"KECCAK1600_ASM" -D"RC4_ASM" -D"MD5_ASM" -D"AESNI_ASM" -D"VPAES_ASM" -D"GHASH_ASM" -D"ECP_NISTZ256_ASM" -D"X25519_ASM" -D"POLY1305_ASM" -D"OPENSSLDIR=\"C:\\Users\\malas\\.conan\\data\\openssl\\1.1.1h\\_\\_\\package\\402cf37ea699030d7c8abcaa214949592ce22e9b\\res\"" -D"ENGINESDIR=\"C:\\Users\\malas\\.conan\\data\\openssl\\1.1.1h\\_\\_\\package\\402cf37ea699030d7c8abcaa214949592ce22e9b\\lib\\engines-1_1\"" -D"OPENSSL_SYS_WIN32" -D"WIN32_LEAN_AND_MEAN" -D"UNICODE" -D"_UNICODE" -D"_CRT_SECURE_NO_DEPRECATE" -D"_WINSOCK_DEPRECATED_NO_WARNINGS" -D"DEBUG" -D"_DEBUG" -I"C:/.conan/1be2a02/1/include"  -c /Foapps\app_rand.obj "apps\app_rand.c"
app_rand.c
        cl  /Zi /Fdossl_static.pdb /MD /Zl /Gs0 /GF /Gy -Zi -Ob0 -Od -MD /W3 /wd4090 /nologo /Od /I "." /I "include" -D"L_ENDIAN" -D"OPENSSL_PIC" -D"OPENSSL_CPUID_OBJ" -D"OPENSSL_IA32_SSE2" -D"OPENSSL_BN_ASM_MONT" -D"OPENSSL_BN_ASM_MONT5" -D"OPENSSL_BN_ASM_GF2m" -D"SHA1_ASM" -D"SHA256_ASM" -D"SHA512_ASM" -D"KECCAK1600_ASM" -D"RC4_ASM" -D"MD5_ASM" -D"AESNI_ASM" -D"VPAES_ASM" -D"GHASH_ASM" -D"ECP_NISTZ256_ASM" -D"X25519_ASM" -D"POLY1305_ASM" -D"OPENSSLDIR=\"C:\\Users\\malas\\.conan\\data\\openssl\\1.1.1h\\_\\_\\package\\402cf37ea699030d7c8abcaa214949592ce22e9b\\res\"" -D"ENGINESDIR=\"C:\\Users\\malas\\.conan\\data\\openssl\\1.1.1h\\_\\_\\package\\402cf37ea699030d7c8abcaa214949592ce22e9b\\lib\\engines-1_1\"" -D"OPENSSL_SYS_WIN32" -D"WIN32_LEAN_AND_MEAN" -D"UNICODE" -D"_UNICODE" -D"_CRT_SECURE_NO_DEPRECATE" -D"_WINSOCK_DEPRECATED_NO_WARNINGS" -D"DEBUG" -D"_DEBUG" -I"C:/.conan/1be2a02/1/include"  /Zs /showIncludes "apps\app_rand.c" 2>&1 > apps\app_rand.d
        cl  /Zi /Fdossl_static.pdb /MD /Zl /Gs0 /GF /Gy -Zi -Ob0 -Od -MD /W3 /wd4090 /nologo /Od /I "." /I "include" -D"L_ENDIAN" -D"OPENSSL_PIC" -D"OPENSSL_CPUID_OBJ" -D"OPENSSL_IA32_SSE2" -D"OPENSSL_BN_ASM_MONT" -D"OPENSSL_BN_ASM_MONT5" -D"OPENSSL_BN_ASM_GF2m" -D"SHA1_ASM" -D"SHA256_ASM" -D"SHA512_ASM" -D"KECCAK1600_ASM" -D"RC4_ASM" -D"MD5_ASM" -D"AESNI_ASM" -D"VPAES_ASM" -D"GHASH_ASM" -D"ECP_NISTZ256_ASM" -D"X25519_ASM" -D"POLY1305_ASM" -D"OPENSSLDIR=\"C:\\Users\\malas\\.conan\\data\\openssl\\1.1.1h\\_\\_\\package\\402cf37ea699030d7c8abcaa214949592ce22e9b\\res\"" -D"ENGINESDIR=\"C:\\Users\\malas\\.conan\\data\\openssl\\1.1.1h\\_\\_\\package\\402cf37ea699030d7c8abcaa214949592ce22e9b\\lib\\engines-1_1\"" -D"OPENSSL_SYS_WIN32" -D"WIN32_LEAN_AND_MEAN" -D"UNICODE" -D"_UNICODE" -D"_CRT_SECURE_NO_DEPRECATE" -D"_WINSOCK_DEPRECATED_NO_WARNINGS" -D"DEBUG" -D"_DEBUG" -I"C:/.conan/1be2a02/1/include"  -c /Foapps\apps.obj "apps\apps.c"
apps.c
apps\apps.c(1971): warning C4267: 'function': conversion from 'size_t' to 'int', possible loss of data
apps\apps.c(2142): warning C4267: 'function': conversion from 'size_t' to 'int', possible loss of data
apps\apps.c(2142): warning C4267: 'function': conversion from 'size_t' to 'int', possible loss of data
apps\apps.c(2147): warning C4267: 'function': conversion from 'size_t' to 'int', possible loss of data
apps\apps.c(2147): warning C4267: 'function': conversion from 'size_t' to 'int', possible loss of data
apps\apps.c(2368): warning C4267: 'function': conversion from 'size_t' to 'int', possible loss of data
        cl  /Zi /Fdossl_static.pdb /MD /Zl /Gs0 /GF /Gy -Zi -Ob0 -Od -MD /W3 /wd4090 /nologo /Od /I "." /I "include" -D"L_ENDIAN" -D"OPENSSL_PIC" -D"OPENSSL_CPUID_OBJ" -D"OPENSSL_IA32_SSE2" -D"OPENSSL_BN_ASM_MONT" -D"OPENSSL_BN_ASM_MONT5" -D"OPENSSL_BN_ASM_GF2m" -D"SHA1_ASM" -D"SHA256_ASM" -D"SHA512_ASM" -D"KECCAK1600_ASM" -D"RC4_ASM" -D"MD5_ASM" -D"AESNI_ASM" -D"VPAES_ASM" -D"GHASH_ASM" -D"ECP_NISTZ256_ASM" -D"X25519_ASM" -D"POLY1305_ASM" -D"OPENSSLDIR=\"C:\\Users\\malas\\.conan\\data\\openssl\\1.1.1h\\_\\_\\package\\402cf37ea699030d7c8abcaa214949592ce22e9b\\res\"" -D"ENGINESDIR=\"C:\\Users\\malas\\.conan\\data\\openssl\\1.1.1h\\_\\_\\package\\402cf37ea699030d7c8abcaa214949592ce22e9b\\lib\\engines-1_1\"" -D"OPENSSL_SYS_WIN32" -D"WIN32_LEAN_AND_MEAN" -D"UNICODE" -D"_UNICODE" -D"_CRT_SECURE_NO_DEPRECATE" -D"_WINSOCK_DEPRECATED_NO_WARNINGS" -D"DEBUG" -D"_DEBUG" -I"C:/.conan/1be2a02/1/include"  /Zs /showIncludes "apps\apps.c" 2>&1 > apps\apps.d
        cl  /Zi /Fdossl_static.pdb /MD /Zl /Gs0 /GF /Gy -Zi -Ob0 -Od -MD /W3 /wd4090 /nologo /Od /I "." /I "include" -D"L_ENDIAN" -D"OPENSSL_PIC" -D"OPENSSL_CPUID_OBJ" -D"OPENSSL_IA32_SSE2" -D"OPENSSL_BN_ASM_MONT" -D"OPENSSL_BN_ASM_MONT5" -D"OPENSSL_BN_ASM_GF2m" -D"SHA1_ASM" -D"SHA256_ASM" -D"SHA512_ASM" -D"KECCAK1600_ASM" -D"RC4_ASM" -D"MD5_ASM" -D"AESNI_ASM" -D"VPAES_ASM" -D"GHASH_ASM" -D"ECP_NISTZ256_ASM" -D"X25519_ASM" -D"POLY1305_ASM" -D"OPENSSLDIR=\"C:\\Users\\malas\\.conan\\data\\openssl\\1.1.1h\\_\\_\\package\\402cf37ea699030d7c8abcaa214949592ce22e9b\\res\"" -D"ENGINESDIR=\"C:\\Users\\malas\\.conan\\data\\openssl\\1.1.1h\\_\\_\\package\\402cf37ea699030d7c8abcaa214949592ce22e9b\\lib\\engines-1_1\"" -D"OPENSSL_SYS_WIN32" -D"WIN32_LEAN_AND_MEAN" -D"UNICODE" -D"_UNICODE" -D"_CRT_SECURE_NO_DEPRECATE" -D"_WINSOCK_DEPRECATED_NO_WARNINGS" -D"DEBUG" -D"_DEBUG" -I"C:/.conan/1be2a02/1/include"  -c /Foapps\bf_prefix.obj "apps\bf_prefix.c"
bf_prefix.c
        cl  /Zi /Fdossl_static.pdb /MD /Zl /Gs0 /GF /Gy -Zi -Ob0 -Od -MD /W3 /wd4090 /nologo /Od /I "." /I "include" -D"L_ENDIAN" -D"OPENSSL_PIC" -D"OPENSSL_CPUID_OBJ" -D"OPENSSL_IA32_SSE2" -D"OPENSSL_BN_ASM_MONT" -D"OPENSSL_BN_ASM_MONT5" -D"OPENSSL_BN_ASM_GF2m" -D"SHA1_ASM" -D"SHA256_ASM" -D"SHA512_ASM" -D"KECCAK1600_ASM" -D"RC4_ASM" -D"MD5_ASM" -D"AESNI_ASM" -D"VPAES_ASM" -D"GHASH_ASM" -D"ECP_NISTZ256_ASM" -D"X25519_ASM" -D"POLY1305_ASM" -D"OPENSSLDIR=\"C:\\Users\\malas\\.conan\\data\\openssl\\1.1.1h\\_\\_\\package\\402cf37ea699030d7c8abcaa214949592ce22e9b\\res\"" -D"ENGINESDIR=\"C:\\Users\\malas\\.conan\\data\\openssl\\1.1.1h\\_\\_\\package\\402cf37ea699030d7c8abcaa214949592ce22e9b\\lib\\engines-1_1\"" -D"OPENSSL_SYS_WIN32" -D"WIN32_LEAN_AND_MEAN" -D"UNICODE" -D"_UNICODE" -D"_CRT_SECURE_NO_DEPRECATE" -D"_WINSOCK_DEPRECATED_NO_WARNINGS" -D"DEBUG" -D"_DEBUG" -I"C:/.conan/1be2a02/1/include"  /Zs /showIncludes "apps\bf_prefix.c" 2>&1 > apps\bf_prefix.d
        cl  /Zi /Fdossl_static.pdb /MD /Zl /Gs0 /GF /Gy -Zi -Ob0 -Od -MD /W3 /wd4090 /nologo /Od /I "." /I "include" -D"L_ENDIAN" -D"OPENSSL_PIC" -D"OPENSSL_CPUID_OBJ" -D"OPENSSL_IA32_SSE2" -D"OPENSSL_BN_ASM_MONT" -D"OPENSSL_BN_ASM_MONT5" -D"OPENSSL_BN_ASM_GF2m" -D"SHA1_ASM" -D"SHA256_ASM" -D"SHA512_ASM" -D"KECCAK1600_ASM" -D"RC4_ASM" -D"MD5_ASM" -D"AESNI_ASM" -D"VPAES_ASM" -D"GHASH_ASM" -D"ECP_NISTZ256_ASM" -D"X25519_ASM" -D"POLY1305_ASM" -D"OPENSSLDIR=\"C:\\Users\\malas\\.conan\\data\\openssl\\1.1.1h\\_\\_\\package\\402cf37ea699030d7c8abcaa214949592ce22e9b\\res\"" -D"ENGINESDIR=\"C:\\Users\\malas\\.conan\\data\\openssl\\1.1.1h\\_\\_\\package\\402cf37ea699030d7c8abcaa214949592ce22e9b\\lib\\engines-1_1\"" -D"OPENSSL_SYS_WIN32" -D"WIN32_LEAN_AND_MEAN" -D"UNICODE" -D"_UNICODE" -D"_CRT_SECURE_NO_DEPRECATE" -D"_WINSOCK_DEPRECATED_NO_WARNINGS" -D"DEBUG" -D"_DEBUG" -I"C:/.conan/1be2a02/1/include"  -c /Foapps\opt.obj "apps\opt.c"
opt.c
        cl  /Zi /Fdossl_static.pdb /MD /Zl /Gs0 /GF /Gy -Zi -Ob0 -Od -MD /W3 /wd4090 /nologo /Od /I "." /I "include" -D"L_ENDIAN" -D"OPENSSL_PIC" -D"OPENSSL_CPUID_OBJ" -D"OPENSSL_IA32_SSE2" -D"OPENSSL_BN_ASM_MONT" -D"OPENSSL_BN_ASM_MONT5" -D"OPENSSL_BN_ASM_GF2m" -D"SHA1_ASM" -D"SHA256_ASM" -D"SHA512_ASM" -D"KECCAK1600_ASM" -D"RC4_ASM" -D"MD5_ASM" -D"AESNI_ASM" -D"VPAES_ASM" -D"GHASH_ASM" -D"ECP_NISTZ256_ASM" -D"X25519_ASM" -D"POLY1305_ASM" -D"OPENSSLDIR=\"C:\\Users\\malas\\.conan\\data\\openssl\\1.1.1h\\_\\_\\package\\402cf37ea699030d7c8abcaa214949592ce22e9b\\res\"" -D"ENGINESDIR=\"C:\\Users\\malas\\.conan\\data\\openssl\\1.1.1h\\_\\_\\package\\402cf37ea699030d7c8abcaa214949592ce22e9b\\lib\\engines-1_1\"" -D"OPENSSL_SYS_WIN32" -D"WIN32_LEAN_AND_MEAN" -D"UNICODE" -D"_UNICODE" -D"_CRT_SECURE_NO_DEPRECATE" -D"_WINSOCK_DEPRECATED_NO_WARNINGS" -D"DEBUG" -D"_DEBUG" -I"C:/.conan/1be2a02/1/include"  /Zs /showIncludes "apps\opt.c" 2>&1 > apps\opt.d
        cl  /Zi /Fdossl_static.pdb /MD /Zl /Gs0 /GF /Gy -Zi -Ob0 -Od -MD /W3 /wd4090 /nologo /Od /I "." /I "include" -D"L_ENDIAN" -D"OPENSSL_PIC" -D"OPENSSL_CPUID_OBJ" -D"OPENSSL_IA32_SSE2" -D"OPENSSL_BN_ASM_MONT" -D"OPENSSL_BN_ASM_MONT5" -D"OPENSSL_BN_ASM_GF2m" -D"SHA1_ASM" -D"SHA256_ASM" -D"SHA512_ASM" -D"KECCAK1600_ASM" -D"RC4_ASM" -D"MD5_ASM" -D"AESNI_ASM" -D"VPAES_ASM" -D"GHASH_ASM" -D"ECP_NISTZ256_ASM" -D"X25519_ASM" -D"POLY1305_ASM" -D"OPENSSLDIR=\"C:\\Users\\malas\\.conan\\data\\openssl\\1.1.1h\\_\\_\\package\\402cf37ea699030d7c8abcaa214949592ce22e9b\\res\"" -D"ENGINESDIR=\"C:\\Users\\malas\\.conan\\data\\openssl\\1.1.1h\\_\\_\\package\\402cf37ea699030d7c8abcaa214949592ce22e9b\\lib\\engines-1_1\"" -D"OPENSSL_SYS_WIN32" -D"WIN32_LEAN_AND_MEAN" -D"UNICODE" -D"_UNICODE" -D"_CRT_SECURE_NO_DEPRECATE" -D"_WINSOCK_DEPRECATED_NO_WARNINGS" -D"DEBUG" -D"_DEBUG" -I"C:/.conan/1be2a02/1/include"  -c /Foapps\s_cb.obj "apps\s_cb.c"
s_cb.c
apps\s_cb.c(785): warning C4267: 'function': conversion from 'size_t' to 'int', possible loss of data
apps\s_cb.c(829): warning C4267: 'function': conversion from 'size_t' to 'unsigned int', possible loss of data
        cl  /Zi /Fdossl_static.pdb /MD /Zl /Gs0 /GF /Gy -Zi -Ob0 -Od -MD /W3 /wd4090 /nologo /Od /I "." /I "include" -D"L_ENDIAN" -D"OPENSSL_PIC" -D"OPENSSL_CPUID_OBJ" -D"OPENSSL_IA32_SSE2" -D"OPENSSL_BN_ASM_MONT" -D"OPENSSL_BN_ASM_MONT5" -D"OPENSSL_BN_ASM_GF2m" -D"SHA1_ASM" -D"SHA256_ASM" -D"SHA512_ASM" -D"KECCAK1600_ASM" -D"RC4_ASM" -D"MD5_ASM" -D"AESNI_ASM" -D"VPAES_ASM" -D"GHASH_ASM" -D"ECP_NISTZ256_ASM" -D"X25519_ASM" -D"POLY1305_ASM" -D"OPENSSLDIR=\"C:\\Users\\malas\\.conan\\data\\openssl\\1.1.1h\\_\\_\\package\\402cf37ea699030d7c8abcaa214949592ce22e9b\\res\"" -D"ENGINESDIR=\"C:\\Users\\malas\\.conan\\data\\openssl\\1.1.1h\\_\\_\\package\\402cf37ea699030d7c8abcaa214949592ce22e9b\\lib\\engines-1_1\"" -D"OPENSSL_SYS_WIN32" -D"WIN32_LEAN_AND_MEAN" -D"UNICODE" -D"_UNICODE" -D"_CRT_SECURE_NO_DEPRECATE" -D"_WINSOCK_DEPRECATED_NO_WARNINGS" -D"DEBUG" -D"_DEBUG" -I"C:/.conan/1be2a02/1/include"  /Zs /showIncludes "apps\s_cb.c" 2>&1 > apps\s_cb.d
        cl  /Zi /Fdossl_static.pdb /MD /Zl /Gs0 /GF /Gy -Zi -Ob0 -Od -MD /W3 /wd4090 /nologo /Od /I "." /I "include" -D"L_ENDIAN" -D"OPENSSL_PIC" -D"OPENSSL_CPUID_OBJ" -D"OPENSSL_IA32_SSE2" -D"OPENSSL_BN_ASM_MONT" -D"OPENSSL_BN_ASM_MONT5" -D"OPENSSL_BN_ASM_GF2m" -D"SHA1_ASM" -D"SHA256_ASM" -D"SHA512_ASM" -D"KECCAK1600_ASM" -D"RC4_ASM" -D"MD5_ASM" -D"AESNI_ASM" -D"VPAES_ASM" -D"GHASH_ASM" -D"ECP_NISTZ256_ASM" -D"X25519_ASM" -D"POLY1305_ASM" -D"OPENSSLDIR=\"C:\\Users\\malas\\.conan\\data\\openssl\\1.1.1h\\_\\_\\package\\402cf37ea699030d7c8abcaa214949592ce22e9b\\res\"" -D"ENGINESDIR=\"C:\\Users\\malas\\.conan\\data\\openssl\\1.1.1h\\_\\_\\package\\402cf37ea699030d7c8abcaa214949592ce22e9b\\lib\\engines-1_1\"" -D"OPENSSL_SYS_WIN32" -D"WIN32_LEAN_AND_MEAN" -D"UNICODE" -D"_UNICODE" -D"_CRT_SECURE_NO_DEPRECATE" -D"_WINSOCK_DEPRECATED_NO_WARNINGS" -D"DEBUG" -D"_DEBUG" -I"C:/.conan/1be2a02/1/include"  -c /Foapps\s_socket.obj "apps\s_socket.c"
s_socket.c
        cl  /Zi /Fdossl_static.pdb /MD /Zl /Gs0 /GF /Gy -Zi -Ob0 -Od -MD /W3 /wd4090 /nologo /Od /I "." /I "include" -D"L_ENDIAN" -D"OPENSSL_PIC" -D"OPENSSL_CPUID_OBJ" -D"OPENSSL_IA32_SSE2" -D"OPENSSL_BN_ASM_MONT" -D"OPENSSL_BN_ASM_MONT5" -D"OPENSSL_BN_ASM_GF2m" -D"SHA1_ASM" -D"SHA256_ASM" -D"SHA512_ASM" -D"KECCAK1600_ASM" -D"RC4_ASM" -D"MD5_ASM" -D"AESNI_ASM" -D"VPAES_ASM" -D"GHASH_ASM" -D"ECP_NISTZ256_ASM" -D"X25519_ASM" -D"POLY1305_ASM" -D"OPENSSLDIR=\"C:\\Users\\malas\\.conan\\data\\openssl\\1.1.1h\\_\\_\\package\\402cf37ea699030d7c8abcaa214949592ce22e9b\\res\"" -D"ENGINESDIR=\"C:\\Users\\malas\\.conan\\data\\openssl\\1.1.1h\\_\\_\\package\\402cf37ea699030d7c8abcaa214949592ce22e9b\\lib\\engines-1_1\"" -D"OPENSSL_SYS_WIN32" -D"WIN32_LEAN_AND_MEAN" -D"UNICODE" -D"_UNICODE" -D"_CRT_SECURE_NO_DEPRECATE" -D"_WINSOCK_DEPRECATED_NO_WARNINGS" -D"DEBUG" -D"_DEBUG" -I"C:/.conan/1be2a02/1/include"  /Zs /showIncludes "apps\s_socket.c" 2>&1 > apps\s_socket.d
        cl  /Zi /Fdossl_static.pdb /MD /Zl /Gs0 /GF /Gy -Zi -Ob0 -Od -MD /W3 /wd4090 /nologo /Od /I "." /I "include" -D"L_ENDIAN" -D"OPENSSL_PIC" -D"OPENSSL_CPUID_OBJ" -D"OPENSSL_IA32_SSE2" -D"OPENSSL_BN_ASM_MONT" -D"OPENSSL_BN_ASM_MONT5" -D"OPENSSL_BN_ASM_GF2m" -D"SHA1_ASM" -D"SHA256_ASM" -D"SHA512_ASM" -D"KECCAK1600_ASM" -D"RC4_ASM" -D"MD5_ASM" -D"AESNI_ASM" -D"VPAES_ASM" -D"GHASH_ASM" -D"ECP_NISTZ256_ASM" -D"X25519_ASM" -D"POLY1305_ASM" -D"OPENSSLDIR=\"C:\\Users\\malas\\.conan\\data\\openssl\\1.1.1h\\_\\_\\package\\402cf37ea699030d7c8abcaa214949592ce22e9b\\res\"" -D"ENGINESDIR=\"C:\\Users\\malas\\.conan\\data\\openssl\\1.1.1h\\_\\_\\package\\402cf37ea699030d7c8abcaa214949592ce22e9b\\lib\\engines-1_1\"" -D"OPENSSL_SYS_WIN32" -D"WIN32_LEAN_AND_MEAN" -D"UNICODE" -D"_UNICODE" -D"_CRT_SECURE_NO_DEPRECATE" -D"_WINSOCK_DEPRECATED_NO_WARNINGS" -D"DEBUG" -D"_DEBUG" -I"C:/.conan/1be2a02/1/include"  -c /Foapps\win32_init.obj "apps\win32_init.c"
win32_init.c
apps\win32_init.c(162): warning C4267: '=': conversion from 'size_t' to 'int', possible loss of data
apps\win32_init.c(216): warning C4244: '=': conversion from '__int64' to 'int', possible loss of data
apps\win32_init.c(226): warning C4244: '=': conversion from '__int64' to 'int', possible loss of data
apps\win32_init.c(251): warning C4244: '=': conversion from '__int64' to 'int', possible loss of data
        cl  /Zi /Fdossl_static.pdb /MD /Zl /Gs0 /GF /Gy -Zi -Ob0 -Od -MD /W3 /wd4090 /nologo /Od /I "." /I "include" -D"L_ENDIAN" -D"OPENSSL_PIC" -D"OPENSSL_CPUID_OBJ" -D"OPENSSL_IA32_SSE2" -D"OPENSSL_BN_ASM_MONT" -D"OPENSSL_BN_ASM_MONT5" -D"OPENSSL_BN_ASM_GF2m" -D"SHA1_ASM" -D"SHA256_ASM" -D"SHA512_ASM" -D"KECCAK1600_ASM" -D"RC4_ASM" -D"MD5_ASM" -D"AESNI_ASM" -D"VPAES_ASM" -D"GHASH_ASM" -D"ECP_NISTZ256_ASM" -D"X25519_ASM" -D"POLY1305_ASM" -D"OPENSSLDIR=\"C:\\Users\\malas\\.conan\\data\\openssl\\1.1.1h\\_\\_\\package\\402cf37ea699030d7c8abcaa214949592ce22e9b\\res\"" -D"ENGINESDIR=\"C:\\Users\\malas\\.conan\\data\\openssl\\1.1.1h\\_\\_\\package\\402cf37ea699030d7c8abcaa214949592ce22e9b\\lib\\engines-1_1\"" -D"OPENSSL_SYS_WIN32" -D"WIN32_LEAN_AND_MEAN" -D"UNICODE" -D"_UNICODE" -D"_CRT_SECURE_NO_DEPRECATE" -D"_WINSOCK_DEPRECATED_NO_WARNINGS" -D"DEBUG" -D"_DEBUG" -I"C:/.conan/1be2a02/1/include"  /Zs /showIncludes "apps\win32_init.c" 2>&1 > apps\win32_init.d
        lib /nologo /out:apps\libapps.lib @C:\Users\malas\AppData\Local\Temp\nmC67F.tmp
        cl  /Zi /Fdossl_static.pdb /MD /Zl /Gs0 /GF /Gy -Zi -Ob0 -Od -MD /W3 /wd4090 /nologo /Od /I "." /I "include" -D"L_ENDIAN" -D"OPENSSL_PIC" -D"OPENSSL_CPUID_OBJ" -D"OPENSSL_IA32_SSE2" -D"OPENSSL_BN_ASM_MONT" -D"OPENSSL_BN_ASM_MONT5" -D"OPENSSL_BN_ASM_GF2m" -D"SHA1_ASM" -D"SHA256_ASM" -D"SHA512_ASM" -D"KECCAK1600_ASM" -D"RC4_ASM" -D"MD5_ASM" -D"AESNI_ASM" -D"VPAES_ASM" -D"GHASH_ASM" -D"ECP_NISTZ256_ASM" -D"X25519_ASM" -D"POLY1305_ASM" -D"OPENSSLDIR=\"C:\\Users\\malas\\.conan\\data\\openssl\\1.1.1h\\_\\_\\package\\402cf37ea699030d7c8abcaa214949592ce22e9b\\res\"" -D"ENGINESDIR=\"C:\\Users\\malas\\.conan\\data\\openssl\\1.1.1h\\_\\_\\package\\402cf37ea699030d7c8abcaa214949592ce22e9b\\lib\\engines-1_1\"" -D"OPENSSL_SYS_WIN32" -D"WIN32_LEAN_AND_MEAN" -D"UNICODE" -D"_UNICODE" -D"_CRT_SECURE_NO_DEPRECATE" -D"_WINSOCK_DEPRECATED_NO_WARNINGS" -D"DEBUG" -D"_DEBUG" -I"C:/.conan/1be2a02/1/include"  -c /Focrypto\aes\aes_cbc.obj "crypto\aes\aes_cbc.c"
aes_cbc.c
        cl  /Zi /Fdossl_static.pdb /MD /Zl /Gs0 /GF /Gy -Zi -Ob0 -Od -MD /W3 /wd4090 /nologo /Od /I "." /I "include" -D"L_ENDIAN" -D"OPENSSL_PIC" -D"OPENSSL_CPUID_OBJ" -D"OPENSSL_IA32_SSE2" -D"OPENSSL_BN_ASM_MONT" -D"OPENSSL_BN_ASM_MONT5" -D"OPENSSL_BN_ASM_GF2m" -D"SHA1_ASM" -D"SHA256_ASM" -D"SHA512_ASM" -D"KECCAK1600_ASM" -D"RC4_ASM" -D"MD5_ASM" -D"AESNI_ASM" -D"VPAES_ASM" -D"GHASH_ASM" -D"ECP_NISTZ256_ASM" -D"X25519_ASM" -D"POLY1305_ASM" -D"OPENSSLDIR=\"C:\\Users\\malas\\.conan\\data\\openssl\\1.1.1h\\_\\_\\package\\402cf37ea699030d7c8abcaa214949592ce22e9b\\res\"" -D"ENGINESDIR=\"C:\\Users\\malas\\.conan\\data\\openssl\\1.1.1h\\_\\_\\package\\402cf37ea699030d7c8abcaa214949592ce22e9b\\lib\\engines-1_1\"" -D"OPENSSL_SYS_WIN32" -D"WIN32_LEAN_AND_MEAN" -D"UNICODE" -D"_UNICODE" -D"_CRT_SECURE_NO_DEPRECATE" -D"_WINSOCK_DEPRECATED_NO_WARNINGS" -D"DEBUG" -D"_DEBUG" -I"C:/.conan/1be2a02/1/include"  /Zs /showIncludes "crypto\aes\aes_cbc.c" 2>&1 > crypto\aes\aes_cbc.d
        cl  /Zi /Fdossl_static.pdb /MD /Zl /Gs0 /GF /Gy -Zi -Ob0 -Od -MD /W3 /wd4090 /nologo /Od /I "." /I "include" -D"L_ENDIAN" -D"OPENSSL_PIC" -D"OPENSSL_CPUID_OBJ" -D"OPENSSL_IA32_SSE2" -D"OPENSSL_BN_ASM_MONT" -D"OPENSSL_BN_ASM_MONT5" -D"OPENSSL_BN_ASM_GF2m" -D"SHA1_ASM" -D"SHA256_ASM" -D"SHA512_ASM" -D"KECCAK1600_ASM" -D"RC4_ASM" -D"MD5_ASM" -D"AESNI_ASM" -D"VPAES_ASM" -D"GHASH_ASM" -D"ECP_NISTZ256_ASM" -D"X25519_ASM" -D"POLY1305_ASM" -D"OPENSSLDIR=\"C:\\Users\\malas\\.conan\\data\\openssl\\1.1.1h\\_\\_\\package\\402cf37ea699030d7c8abcaa214949592ce22e9b\\res\"" -D"ENGINESDIR=\"C:\\Users\\malas\\.conan\\data\\openssl\\1.1.1h\\_\\_\\package\\402cf37ea699030d7c8abcaa214949592ce22e9b\\lib\\engines-1_1\"" -D"OPENSSL_SYS_WIN32" -D"WIN32_LEAN_AND_MEAN" -D"UNICODE" -D"_UNICODE" -D"_CRT_SECURE_NO_DEPRECATE" -D"_WINSOCK_DEPRECATED_NO_WARNINGS" -D"DEBUG" -D"_DEBUG" -I"C:/.conan/1be2a02/1/include"  -c /Focrypto\aes\aes_cfb.obj "crypto\aes\aes_cfb.c"
aes_cfb.c
        cl  /Zi /Fdossl_static.pdb /MD /Zl /Gs0 /GF /Gy -Zi -Ob0 -Od -MD /W3 /wd4090 /nologo /Od /I "." /I "include" -D"L_ENDIAN" -D"OPENSSL_PIC" -D"OPENSSL_CPUID_OBJ" -D"OPENSSL_IA32_SSE2" -D"OPENSSL_BN_ASM_MONT" -D"OPENSSL_BN_ASM_MONT5" -D"OPENSSL_BN_ASM_GF2m" -D"SHA1_ASM" -D"SHA256_ASM" -D"SHA512_ASM" -D"KECCAK1600_ASM" -D"RC4_ASM" -D"MD5_ASM" -D"AESNI_ASM" -D"VPAES_ASM" -D"GHASH_ASM" -D"ECP_NISTZ256_ASM" -D"X25519_ASM" -D"POLY1305_ASM" -D"OPENSSLDIR=\"C:\\Users\\malas\\.conan\\data\\openssl\\1.1.1h\\_\\_\\package\\402cf37ea699030d7c8abcaa214949592ce22e9b\\res\"" -D"ENGINESDIR=\"C:\\Users\\malas\\.conan\\data\\openssl\\1.1.1h\\_\\_\\package\\402cf37ea699030d7c8abcaa214949592ce22e9b\\lib\\engines-1_1\"" -D"OPENSSL_SYS_WIN32" -D"WIN32_LEAN_AND_MEAN" -D"UNICODE" -D"_UNICODE" -D"_CRT_SECURE_NO_DEPRECATE" -D"_WINSOCK_DEPRECATED_NO_WARNINGS" -D"DEBUG" -D"_DEBUG" -I"C:/.conan/1be2a02/1/include"  /Zs /showIncludes "crypto\aes\aes_cfb.c" 2>&1 > crypto\aes\aes_cfb.d
        cl  /Zi /Fdossl_static.pdb /MD /Zl /Gs0 /GF /Gy -Zi -Ob0 -Od -MD /W3 /wd4090 /nologo /Od /I "." /I "include" -D"L_ENDIAN" -D"OPENSSL_PIC" -D"OPENSSL_CPUID_OBJ" -D"OPENSSL_IA32_SSE2" -D"OPENSSL_BN_ASM_MONT" -D"OPENSSL_BN_ASM_MONT5" -D"OPENSSL_BN_ASM_GF2m" -D"SHA1_ASM" -D"SHA256_ASM" -D"SHA512_ASM" -D"KECCAK1600_ASM" -D"RC4_ASM" -D"MD5_ASM" -D"AESNI_ASM" -D"VPAES_ASM" -D"GHASH_ASM" -D"ECP_NISTZ256_ASM" -D"X25519_ASM" -D"POLY1305_ASM" -D"OPENSSLDIR=\"C:\\Users\\malas\\.conan\\data\\openssl\\1.1.1h\\_\\_\\package\\402cf37ea699030d7c8abcaa214949592ce22e9b\\res\"" -D"ENGINESDIR=\"C:\\Users\\malas\\.conan\\data\\openssl\\1.1.1h\\_\\_\\package\\402cf37ea699030d7c8abcaa214949592ce22e9b\\lib\\engines-1_1\"" -D"OPENSSL_SYS_WIN32" -D"WIN32_LEAN_AND_MEAN" -D"UNICODE" -D"_UNICODE" -D"_CRT_SECURE_NO_DEPRECATE" -D"_WINSOCK_DEPRECATED_NO_WARNINGS" -D"DEBUG" -D"_DEBUG" -I"C:/.conan/1be2a02/1/include"  -c /Focrypto\aes\aes_core.obj "crypto\aes\aes_core.c"
aes_core.c
        cl  /Zi /Fdossl_static.pdb /MD /Zl /Gs0 /GF /Gy -Zi -Ob0 -Od -MD /W3 /wd4090 /nologo /Od /I "." /I "include" -D"L_ENDIAN" -D"OPENSSL_PIC" -D"OPENSSL_CPUID_OBJ" -D"OPENSSL_IA32_SSE2" -D"OPENSSL_BN_ASM_MONT" -D"OPENSSL_BN_ASM_MONT5" -D"OPENSSL_BN_ASM_GF2m" -D"SHA1_ASM" -D"SHA256_ASM" -D"SHA512_ASM" -D"KECCAK1600_ASM" -D"RC4_ASM" -D"MD5_ASM" -D"AESNI_ASM" -D"VPAES_ASM" -D"GHASH_ASM" -D"ECP_NISTZ256_ASM" -D"X25519_ASM" -D"POLY1305_ASM" -D"OPENSSLDIR=\"C:\\Users\\malas\\.conan\\data\\openssl\\1.1.1h\\_\\_\\package\\402cf37ea699030d7c8abcaa214949592ce22e9b\\res\"" -D"ENGINESDIR=\"C:\\Users\\malas\\.conan\\data\\openssl\\1.1.1h\\_\\_\\package\\402cf37ea699030d7c8abcaa214949592ce22e9b\\lib\\engines-1_1\"" -D"OPENSSL_SYS_WIN32" -D"WIN32_LEAN_AND_MEAN" -D"UNICODE" -D"_UNICODE" -D"_CRT_SECURE_NO_DEPRECATE" -D"_WINSOCK_DEPRECATED_NO_WARNINGS" -D"DEBUG" -D"_DEBUG" -I"C:/.conan/1be2a02/1/include"  /Zs /showIncludes "crypto\aes\aes_core.c" 2>&1 > crypto\aes\aes_core.d
        cl  /Zi /Fdossl_static.pdb /MD /Zl /Gs0 /GF /Gy -Zi -Ob0 -Od -MD /W3 /wd4090 /nologo /Od /I "." /I "include" -D"L_ENDIAN" -D"OPENSSL_PIC" -D"OPENSSL_CPUID_OBJ" -D"OPENSSL_IA32_SSE2" -D"OPENSSL_BN_ASM_MONT" -D"OPENSSL_BN_ASM_MONT5" -D"OPENSSL_BN_ASM_GF2m" -D"SHA1_ASM" -D"SHA256_ASM" -D"SHA512_ASM" -D"KECCAK1600_ASM" -D"RC4_ASM" -D"MD5_ASM" -D"AESNI_ASM" -D"VPAES_ASM" -D"GHASH_ASM" -D"ECP_NISTZ256_ASM" -D"X25519_ASM" -D"POLY1305_ASM" -D"OPENSSLDIR=\"C:\\Users\\malas\\.conan\\data\\openssl\\1.1.1h\\_\\_\\package\\402cf37ea699030d7c8abcaa214949592ce22e9b\\res\"" -D"ENGINESDIR=\"C:\\Users\\malas\\.conan\\data\\openssl\\1.1.1h\\_\\_\\package\\402cf37ea699030d7c8abcaa214949592ce22e9b\\lib\\engines-1_1\"" -D"OPENSSL_SYS_WIN32" -D"WIN32_LEAN_AND_MEAN" -D"UNICODE" -D"_UNICODE" -D"_CRT_SECURE_NO_DEPRECATE" -D"_WINSOCK_DEPRECATED_NO_WARNINGS" -D"DEBUG" -D"_DEBUG" -I"C:/.conan/1be2a02/1/include"  -c /Focrypto\aes\aes_ecb.obj "crypto\aes\aes_ecb.c"
aes_ecb.c
        cl  /Zi /Fdossl_static.pdb /MD /Zl /Gs0 /GF /Gy -Zi -Ob0 -Od -MD /W3 /wd4090 /nologo /Od /I "." /I "include" -D"L_ENDIAN" -D"OPENSSL_PIC" -D"OPENSSL_CPUID_OBJ" -D"OPENSSL_IA32_SSE2" -D"OPENSSL_BN_ASM_MONT" -D"OPENSSL_BN_ASM_MONT5" -D"OPENSSL_BN_ASM_GF2m" -D"SHA1_ASM" -D"SHA256_ASM" -D"SHA512_ASM" -D"KECCAK1600_ASM" -D"RC4_ASM" -D"MD5_ASM" -D"AESNI_ASM" -D"VPAES_ASM" -D"GHASH_ASM" -D"ECP_NISTZ256_ASM" -D"X25519_ASM" -D"POLY1305_ASM" -D"OPENSSLDIR=\"C:\\Users\\malas\\.conan\\data\\openssl\\1.1.1h\\_\\_\\package\\402cf37ea699030d7c8abcaa214949592ce22e9b\\res\"" -D"ENGINESDIR=\"C:\\Users\\malas\\.conan\\data\\openssl\\1.1.1h\\_\\_\\package\\402cf37ea699030d7c8abcaa214949592ce22e9b\\lib\\engines-1_1\"" -D"OPENSSL_SYS_WIN32" -D"WIN32_LEAN_AND_MEAN" -D"UNICODE" -D"_UNICODE" -D"_CRT_SECURE_NO_DEPRECATE" -D"_WINSOCK_DEPRECATED_NO_WARNINGS" -D"DEBUG" -D"_DEBUG" -I"C:/.conan/1be2a02/1/include"  /Zs /showIncludes "crypto\aes\aes_ecb.c" 2>&1 > crypto\aes\aes_ecb.d
        cl  /Zi /Fdossl_static.pdb /MD /Zl /Gs0 /GF /Gy -Zi -Ob0 -Od -MD /W3 /wd4090 /nologo /Od /I "." /I "include" -D"L_ENDIAN" -D"OPENSSL_PIC" -D"OPENSSL_CPUID_OBJ" -D"OPENSSL_IA32_SSE2" -D"OPENSSL_BN_ASM_MONT" -D"OPENSSL_BN_ASM_MONT5" -D"OPENSSL_BN_ASM_GF2m" -D"SHA1_ASM" -D"SHA256_ASM" -D"SHA512_ASM" -D"KECCAK1600_ASM" -D"RC4_ASM" -D"MD5_ASM" -D"AESNI_ASM" -D"VPAES_ASM" -D"GHASH_ASM" -D"ECP_NISTZ256_ASM" -D"X25519_ASM" -D"POLY1305_ASM" -D"OPENSSLDIR=\"C:\\Users\\malas\\.conan\\data\\openssl\\1.1.1h\\_\\_\\package\\402cf37ea699030d7c8abcaa214949592ce22e9b\\res\"" -D"ENGINESDIR=\"C:\\Users\\malas\\.conan\\data\\openssl\\1.1.1h\\_\\_\\package\\402cf37ea699030d7c8abcaa214949592ce22e9b\\lib\\engines-1_1\"" -D"OPENSSL_SYS_WIN32" -D"WIN32_LEAN_AND_MEAN" -D"UNICODE" -D"_UNICODE" -D"_CRT_SECURE_NO_DEPRECATE" -D"_WINSOCK_DEPRECATED_NO_WARNINGS" -D"DEBUG" -D"_DEBUG" -I"C:/.conan/1be2a02/1/include"  -c /Focrypto\aes\aes_ige.obj "crypto\aes\aes_ige.c"
aes_ige.c
        cl  /Zi /Fdossl_static.pdb /MD /Zl /Gs0 /GF /Gy -Zi -Ob0 -Od -MD /W3 /wd4090 /nologo /Od /I "." /I "include" -D"L_ENDIAN" -D"OPENSSL_PIC" -D"OPENSSL_CPUID_OBJ" -D"OPENSSL_IA32_SSE2" -D"OPENSSL_BN_ASM_MONT" -D"OPENSSL_BN_ASM_MONT5" -D"OPENSSL_BN_ASM_GF2m" -D"SHA1_ASM" -D"SHA256_ASM" -D"SHA512_ASM" -D"KECCAK1600_ASM" -D"RC4_ASM" -D"MD5_ASM" -D"AESNI_ASM" -D"VPAES_ASM" -D"GHASH_ASM" -D"ECP_NISTZ256_ASM" -D"X25519_ASM" -D"POLY1305_ASM" -D"OPENSSLDIR=\"C:\\Users\\malas\\.conan\\data\\openssl\\1.1.1h\\_\\_\\package\\402cf37ea699030d7c8abcaa214949592ce22e9b\\res\"" -D"ENGINESDIR=\"C:\\Users\\malas\\.conan\\data\\openssl\\1.1.1h\\_\\_\\package\\402cf37ea699030d7c8abcaa214949592ce22e9b\\lib\\engines-1_1\"" -D"OPENSSL_SYS_WIN32" -D"WIN32_LEAN_AND_MEAN" -D"UNICODE" -D"_UNICODE" -D"_CRT_SECURE_NO_DEPRECATE" -D"_WINSOCK_DEPRECATED_NO_WARNINGS" -D"DEBUG" -D"_DEBUG" -I"C:/.conan/1be2a02/1/include"  /Zs /showIncludes "crypto\aes\aes_ige.c" 2>&1 > crypto\aes\aes_ige.d
        cl  /Zi /Fdossl_static.pdb /MD /Zl /Gs0 /GF /Gy -Zi -Ob0 -Od -MD /W3 /wd4090 /nologo /Od /I "." /I "include" -D"L_ENDIAN" -D"OPENSSL_PIC" -D"OPENSSL_CPUID_OBJ" -D"OPENSSL_IA32_SSE2" -D"OPENSSL_BN_ASM_MONT" -D"OPENSSL_BN_ASM_MONT5" -D"OPENSSL_BN_ASM_GF2m" -D"SHA1_ASM" -D"SHA256_ASM" -D"SHA512_ASM" -D"KECCAK1600_ASM" -D"RC4_ASM" -D"MD5_ASM" -D"AESNI_ASM" -D"VPAES_ASM" -D"GHASH_ASM" -D"ECP_NISTZ256_ASM" -D"X25519_ASM" -D"POLY1305_ASM" -D"OPENSSLDIR=\"C:\\Users\\malas\\.conan\\data\\openssl\\1.1.1h\\_\\_\\package\\402cf37ea699030d7c8abcaa214949592ce22e9b\\res\"" -D"ENGINESDIR=\"C:\\Users\\ENSSL_SYS_WIN32" -D"WIN32_LEAN_AND_MEAN" -D"UNICODE" -D"_UNICODE" -D"_CRT_SECURE_NO_DEPRECATE" -D"_WINSOCK_DEPRECATED_NO_WARNINGS" -D"DEBUG" -D"_DEBUG" -I"C:/.conan/1be2a02/1/include"  -c /Focrypto\aes\aes_misc.obj "crypto\aes\aes_misc.c"
aes_misc.c
        cl  /Zi /Fdossl_static.pdb /MD /Zl /Gs0 /GF /Gy -Zi -Ob0 -Od -MD /W3 /wd4090 /nologo /Od /I "." /I "include" -D"L_ENDIAN" -D"OPENSSL_PIC" -D"OPENSSL_CPUID_OBJ" -D"OPENSSL_IA32_SSE2" -D"OPENSSL_BN_ASM_MONT" -D"OPENSSL_BN_ASM_MONT5" -D"OPENSSL_BN_ASM_GF2m" -D"SHA1_ASM" -D"SHA256_ASM" -D"SHA512_ASM" -D"KECCAK1600_ASM" -D"RC4_ASM" -D"MD5_ASM" -D"AESNI_ASM" -D"VPAES_ASM" -D"GHASH_ASM" -D"ECP_NISTZ256_ASM" -D"X25519_ASM" -D"POLY1305_ASM" -D"OPENSSLDIR=\"C:\\Users\\malas\\.conan\\data\\openssl\\1.1.1h\\_\\_\\package\\402cf37ea699030d7c8abcaa214949592ce22e9b\\res\"" -D"ENGINESDIR=\"C:\\Users\\malas\\.conan\\data\\openssl\\1.1.1h\\_\\_\\package\\402cf37ea699030d7c8abcaa214949592ce22e9b\\lib\\engines-1_1\"" -D"OPENSSL_SYS_WIN32" -D"WIN32_LEAN_AND_MEAN" -D"UNICODE" -D"_UNICODE" -D"_CRT_SECURE_NO_DEPRECATE" -D"_WINSOCK_DEPRECATED_NO_WARNINGS" -D"DEBUG" -D"_DEBUG" -I"C:/.conan/1be2a02/1/include"  /Zs /showIncludes "crypto\aes\aes_misc.c" 2>&1 > crypto\aes\aes_misc.d
        cl  /Zi /Fdossl_static.pdb /MD /Zl /Gs0 /GF /Gy -Zi -Ob0 -Od -MD /W3 /wd4090 /nologo /Od /I "." /I "include" -D"L_ENDIAN" -D"OPENSSL_PIC" -D"OPENSSL_CPUID_OBJ" -D"OPENSSL_IA32_SSE2" -D"OPENSSL_BN_ASM_MONT" -D"OPENSSL_BN_ASM_MONT5" -D"OPENSSL_BN_ASM_GF2m" -D"SHA1_ASM" -D"SHA256_ASM" -D"SHA512_ASM" -D"KECCAK1600_ASM" -D"RC4_ASM" -D"MD5_ASM" -D"AESNI_ASM" -D"VPAES_ASM" -D"GHASH_ASM" -D"ECP_NISTZ256_ASM" -D"X25519_ASM" -D"POLY1305_ASM" -D"OPENSSLDIR=\"C:\\Users\\malas\\.conan\\data\\openssl\\1.1.1h\\_\\_\\package\\402cf37ea699030d7c8abcaa214949592ce22e9b\\res\"" -D"ENGINESDIR=\"C:\\Users\\malas\\.conan\\data\\openssl\\1.1.1h\\_\\_\\package\\402cf37ea699030d7c8abcaa214949592ce22e9b\\lib\\engines-1_1\"" -D"OPENSSL_SYS_WIN32" -D"WIN32_LEAN_AND_MEAN" -D"UNICODE" -D"_UNICODE" -D"_CRT_SECURE_NO_DEPRECATE" -D"_WINSOCK_DEPRECATED_NO_WARNINGS" -D"DEBUG" -D"_DEBUG" -I"C:/.conan/1be2a02/1/include"  -c /Focrypto\aes\aes_ofb.obj "crypto\aes\aes_ofb.c"
aes_ofb.c
        cl  /Zi /Fdossl_static.pdb /MD /Zl /Gs0 /GF /Gy -Zi -Ob0 -Od -MD /W3 /wd4090 /nologo /Od /I "." /I "include" -D"L_ENDIAN" -D"OPENSSL_PIC" -D"OPENSSL_CPUID_OBJ" -D"OPENSSL_IA32_SSE2" -D"OPENSSL_BN_ASM_MONT" -D"OPENSSL_BN_ASM_MONT5" -D"OPENSSL_BN_ASM_GF2m" -D"SHA1_ASM" -D"SHA256_ASM" -D"SHA512_ASM" -D"KECCAK1600_ASM" -D"RC4_ASM" -D"MD5_ASM" -D"AESNI_ASM" -D"VPAES_ASM" -D"GHASH_ASM" -D"ECP_NISTZ256_ASM" -D"X25519_ASM" -D"POLY1305_ASM" -D"OPENSSLDIR=\"C:\\Users\\malas\\.conan\\data\\openssl\\1.1.1h\\_\\_\\package\\402cf37ea699030d7c8abcaa214949592ce22e9b\\res\"" -D"ENGINESDIR=\"C:\\Users\\malas\\.conan\\data\\openssl\\1.1.1h\\_\\_\\package\\402cf37ea699030d7c8abcaa214949592ce22e9b\\lib\\engines-1_1\"" -D"OPENSSL_SYS_WIN32" -D"WIN32_LEAN_AND_MEAN" -D"UNICODE" -D"_UNICODE" -D"_CRT_SECURE_NO_DEPRECATE" -D"_WINSOCK_DEPRECATED_NO_WARNINGS" -D"DEBUG" -D"_DEBUG" -I"C:/.conan/1be2a02/1/include"  /Zs /showIncludes "crypto\aes\aes_ofb.c" 2>&1 > crypto\aes\aes_ofb.d
        cl  /Zi /Fdossl_static.pdb /MD /Zl /Gs0 /GF /Gy -Zi -Ob0 -Od -MD /W3 /wd4090 /nologo /Od /I "." /I "include" -D"L_ENDIAN" -D"OPENSSL_PIC" -D"OPENSSL_CPUID_OBJ" -D"OPENSSL_IA32_SSE2" -D"OPENSSL_BN_ASM_MONT" -D"OPENSSL_BN_ASM_MONT5" -D"OPENSSL_BN_ASM_GF2m" -D"SHA1_ASM" -D"SHA256_ASM" -D"SHA512_ASM" -D"KECCAK1600_ASM" -D"RC4_ASM" -D"MD5_ASM" -D"AESNI_ASM" -D"VPAES_ASM" -D"GHASH_ASM" -D"ECP_NISTZ256_ASM" -D"X25519_ASM" -D"POLY1305_ASM" -D"OPENSSLDIR=\"C:\\Users\\malas\\.conan\\data\\openssl\\1.1.1h\\_\\_\\package\\402cf37ea699030d7c8abcaa214949592ce22e9b\\res\"" -D"ENGINESDIR=\"C:\\Users\\malas\\.conan\\data\\openssl\\1.1.1h\\_\\_\\package\\402cf37ea699030d7c8abcaa214949592ce22e9b\\lib\\engines-1_1\"" -D"OPENSSL_SYS_WIN32" -D"WIN32_LEAN_AND_MEAN" -D"UNICODE" -D"_UNICODE" -D"_CRT_SECURE_NO_DEPRECATE" -D"_WINSOCK_DEPRECATED_NO_WARNINGS" -D"DEBUG" -D"_DEBUG" -I"C:/.conan/1be2a02/1/include"  -c /Focrypto\aes\aes_wrap.obj "crypto\aes\aes_wrap.c"
aes_wrap.c
crypto\aes\aes_wrap.c(18): warning C4267: 'return': conversion from 'size_t' to 'int', possible loss of data
crypto\aes\aes_wrap.c(26): warning C4267: 'return': conversion from 'size_t' to 'int', possible loss of data
        cl  /Zi /Fdossl_static.pdb /MD /Zl /Gs0 /GF /Gy -Zi -Ob0 -Od -MD /W3 /wd4090 /nologo /Od /I "." /I "include" -D"L_ENDIAN" -D"OPENSSL_PIC" -D"OPENSSL_CPUID_OBJ" -D"OPENSSL_IA32_SSE2" -D"OPENSSL_BN_ASM_MONT" -D"OPENSSL_BN_ASM_MONT5" -D"OPENSSL_BN_ASM_GF2m" -D"SHA1_ASM" -D"SHA256_ASM" -D"SHA512_ASM" -D"KECCAK1600_ASM" -D"RC4_ASM" -D"MD5_ASM" -D"AESNI_ASM" -D"VPAES_ASM" -D"GHASH_ASM" -D"ECP_NISTZ256_ASM" -D"X25519_ASM" -D"POLY1305_ASM" -D"OPENSSLDIR=\"C:\\Users\\malas\\.conan\\data\\openssl\\1.1.1h\\_\\_\\package\\402cf37ea699030d7c8abcaa214949592ce22e9b\\res\"" -D"ENGINESDIR=\"C:\\Users\\malas\\.conan\\data\\openssl\\1.1.1h\\_\\_\\package\\402cf37ea699030d7c8abcaa214949592ce22e9b\\lib\\engines-1_1\"" -D"OPENSSL_SYS_WIN32" -D"WIN32_LEAN_AND_MEAN" -D"UNICODE" -D"_UNICODE" -D"_CRT_SECURE_NO_DEPRECATE" -D"_WINSOCK_DEPRECATED_NO_WARNINGS" -D"DEBUG" -D"_DEBUG" -I"C:/.conan/1be2a02/1/include"  /Zs /showIncludes "crypto\aes\aes_wrap.c" 2>&1 > crypto\aes\aes_wrap.d
        set ASM=nasm
        "C:\.conan\1be2a02\1\bin\perl.exe" "crypto\aes\asm\aesni-mb-x86_64.pl" auto crypto\aes\aesni-mb-x86_64.asm
        nasm  -Ox -f win64 -DNEAR -g -o crypto\aes\aesni-mb-x86_64.obj "crypto\aes\aesni-mb-x86_64.asm"
NMAKE : fatal error U1077: 'C:\Users\malas\.conan\data\nasm\2.14\_\_\package\01edd76db8e16db9b38c3cca44ec466a9444c388\bin\nasm.EXE' : return code '0xc0000005'
Stop.
NMAKE : fatal error U1077: '"C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.26.28801\bin\HostX64\x64\nmake.exe"' : return code '0x2'
Stop.
openssl/1.1.1h:
openssl/1.1.1h: ERROR: Package '402cf37ea699030d7c8abcaa214949592ce22e9b' build failed
openssl/1.1.1h: WARN: Build folder C:\Users\malas\.conan\data\openssl\1.1.1h\_\_\build\402cf37ea699030d7c8abcaa214949592ce22e9b
ERROR: openssl/1.1.1h: Error in build() method, line 704
        self._make()
while calling '_make', line 658
        self._run_make()
while calling '_run_make', line 616
        self.run(" ".join(command), win_bash=self._win_bash)
        ConanException: Error 2 while executing nmake
```

</details>

--- a/recipes/openssl/1.x.x/conanfile.py
+++ b/recipes/openssl/1.x.x/conanfile.py
@@ -162,7 +162,7 @@ class OpenSSLConan(ConanFile):
             if not self._win_bash:
                 self.build_requires("strawberryperl/5.30.0.1")
             if not self.options.no_asm and not tools.which("nasm"):
-                self.build_requires("nasm/2.14")
+                self.build_requires("nasm/2.15.05")
         if self._win_bash:
             if "CONAN_BASH_PATH" not in os.environ:
                 self.build_requires("msys2/20200517")

--- a/recipes/openssl/1.x.x/conanfile.py
+++ b/recipes/openssl/1.x.x/conanfile.py
@@ -1,8 +1,7 @@
 import os
 import fnmatch
-import platform
 from functools import total_ordering
-from conans.errors import ConanInvalidConfiguration, ConanException
+from conans.errors import ConanInvalidConfiguration
 from conans import ConanFile, AutoToolsBuildEnvironment, tools
 
 
@@ -92,7 +91,6 @@ class OpenSSLConan(ConanFile):
                "no_aria": [True, False],
                "no_blake2": [True, False],
                "no_camellia": [True, False],
-               "no_cast": [True, False],
                "no_chacha": [True, False],
                "no_cms": [True, False],
                "no_comp": [True, False],
@@ -127,31 +125,31 @@ class OpenSSLConan(ConanFile):
 
     def config_options(self):
         if self._full_version >= "1.1.0":
-              del self.options.no_md2
-              del self.options.no_rc4
-              del self.options.no_rc5
-              del self.options.no_zlib
+            del self.options.no_md2
+            del self.options.no_rc4
+            del self.options.no_rc5
+            del self.options.no_zlib
 
         if self._full_version < "1.1.0":
-              del self.options.no_camellia
-              del self.options.no_cast
-              del self.options.no_cms
-              del self.options.no_comp
-              del self.options.no_dgram
-              del self.options.no_engine
-              del self.options.no_idea
-              del self.options.no_md4
-              del self.options.no_ocsp
-              del self.options.no_srp
-              del self.options.no_ts
-              del self.options.no_whirlpool
+            del self.options.no_camellia
+            del self.options.no_cast
+            del self.options.no_cms
+            del self.options.no_comp
+            del self.options.no_dgram
+            del self.options.no_engine
+            del self.options.no_idea
+            del self.options.no_md4
+            del self.options.no_ocsp
+            del self.options.no_srp
+            del self.options.no_ts
+            del self.options.no_whirlpool
 
         if self._full_version < "1.1.1":
-              del self.options.no_aria
-              del self.options.no_pinshared
-              del self.options.no_sm2
-              del self.options.no_sm3
-              del self.options.no_sm4
+            del self.options.no_aria
+            del self.options.no_pinshared
+            del self.options.no_sm2
+            del self.options.no_sm3
+            del self.options.no_sm4
 
         if self.settings.os != "Windows":
             del self.options.capieng_dialog
@@ -499,9 +497,9 @@ class OpenSSLConan(ConanFile):
                     lib_path = lib_path.replace('\\', '/')
 
                 if zlib_info.shared:
-                  args.append("zlib-dynamic")
+                    args.append("zlib-dynamic")
                 else:
-                  args.append("zlib")
+                    args.append("zlib")
 
                 args.extend(['--with-zlib-include="%s"' % include_path,
                              '--with-zlib-lib="%s"' % lib_path])

